IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcVehicle/Documentation.md

```diff
@@ -0,0 +1 @@
+A vehicle is an _IfcTransportationDevice_ that is used as a means of carrying or transporting something
```

Comment:
```suggestion
A vehicle is a non fixed _IfcTransportationDevice_ that is used as a means of carrying or transporting something (e.g. people or goods).
```

Reply:
Maybe also add as a note (if this is true):
One particular use of _IfcVehicle_ is to model the so called kinematic envelope which for railway traffic is defined as the outline of the space occupied by a rail vehicle when in motion, including the effects of tilt, sway, track cant, etc. in which case it should have an _IfcShapeRepresentation_ with _RepresentationIdentifier_= "Clearance" and _a RepresentationType_ corresponding with the instantiated geometry. Or maybe this belongs elsewhere?

Reply:
Could we add this part in a separate PR directly to the tunnel branch? I would like to keep the current state of bSI separate in order to have a synchronised state. It will definitely not be perfect most likely but as close as possible.

Reply:
I see. Ok!